src/Pelecard/PelecardPaymentStatus.php

```diff
@@ -0,0 +1,423 @@
+<?php
+/**
+ * @file PelecardPaymentStatus class - allows to get payment status message based on status code
+ */
+namespace Pelecard;
+
+class PelecardPaymentStatus {
+
+  /* @var array Payment status messages, english version */
+  const MESSAGES_EN = [
+    '000' => 'Permitted transaction.',
```

Comment:
Thanks (again) for your great contribution!
Why not using associative array here? It will also be easier to use on the get method, don't you think?
```
const $messages = [
  '000' => [
    'en' => 'fsdfd',
    'he' => 'כדכג',
  ],
];
```

Not really sure which one is easier to maintain, but as I know Pelecard - changes are rare and not languages will really be added, so we only enjoy more simplicity on the get method.

Reply:
You're right. I made it like this, because I used Notepad++ and regular expressions to convert whole excel table to single array with one click. I can overwrite it as you want manually, not a big deal.

Reply:
I'll merge it right after you make that change. Thanks!